src/huntsman/pocs/scheduler/field.py

```diff
@@ -0,0 +1,142 @@
+from astroplan import FixedTarget
+from astropy.coordinates import SkyCoord
+
+from panoptes.utils.library import load_module
+from panoptes.pocs.base import PanBase
+from huntsman.pocs.utils import dither
+
+
+class AbstractField(PanBase):
+
+    def __init__(self, name, equinox="J2000", frame="icrs", **kwargs):
```

Comment:
Would it be too challenging to build-in mode where the target is moving? E.g. satellites or Elon's car?

Reply:
That will require some thought but it should be possible. Not for this PR though!